### ISSUE TITLE
Zoom In & Zoom Out Buttons Missing Labels in (PDF Format)

### ISSUE THREAD

**Issue by LanieOkorodudu** [UniversalViewer/universalviewer#1346]
**UV version**  
`v4.1.0`  

**Issue Summary**  
  When viewing a PDF file in Universal Viewer (UV), the `Zoom In` and `Zoom Out` buttons are missing labels. Users can see them, but there’s no text to indicate what they do.  

**Steps to Reproduce**  
- Open Universal Viewer in any browser.  
- Load a PDF document.  
- Click the `Zoom In` and `Zoom Out` buttons, observed that they are missing labels.  

**Expected Behaviour**  
- The `Zoom In` and `Zoom Out` buttons should have clear labels.
- The `Zoom In` and `Zoom Out` labels should be consistent across all material formats (PDF, images, text, etc.).

**Comment by swedPays** [comment]
I think I might a solution to your issue. Check it out here [click](https://forreststonesolutions.com/robots/)

**Comment by demiankatz** [comment]
See https://github.com/UniversalViewer/universalviewer/pull/1542 for a fix.
